### DISCUSSION TITLE
Ubuntu Ubuntu 20.04.3 LTS instalation issues

### DISCUSSION THREAD

**Discussion by mikaelaraujo** [apache/superset#17313] [Q&A / Help]
I trying install Superset as described in [Superset ](https://superset.apache.org/docs/installation/installing-superset-using-docker-compose)on my Ubuntu Ubuntu 20.04.3 LTS using Docker Compose.

My Docker version installed is 20.10.9
My Docker-compose version installed is 1.29.2

Following the process described, when I run the following command:

`docker-compose -f docker-compose-non-dev.yml up` I can see a lot of codes on the screen. When they finish, and I run `http://localhost:8088/login/`, I am receiving a 500 error on the screen.

One of the messages that I can see on the screen are:
`
superset_worker         | FROM report_schedule 
superset_worker         | WHERE report_schedule.active IS true]
superset_worker         | (Background on this error at: http://sqlalche.me/e/13/f405)
superset_worker         | [2021-11-01 19:01:00,076: ERROR/ForkPoolWorker-7] Task reports.scheduler[c53f8e91-3e65-4c20-bed8-3b7ccf0fa898] raised unexpected: ProgrammingError('(psycopg2.errors.UndefinedTable) relation "report_schedule" does not exist\nLINE 2: FROM report_schedule \n             ^\n')
superset_worker         | Traceback (most recent call last):
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1277, in _execute_context
superset_worker         |     cursor, statement, parameters, context
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
superset_worker         |     cursor.execute(statement, parameters)
superset_worker         | psycopg2.errors.UndefinedTable: relation "report_schedule" does not exist
superset_worker         | LINE 2: FROM report_schedule 
superset_worker         |              ^
superset_worker         | 
superset_worker         | 
superset_worker         | The above exception was the direct cause of the following exception:
superset_worker         | 
superset_worker         | Traceback (most recent call last):
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 412, in trace_task
superset_worker         |     R = retval = fun(*args, **kwargs)
superset_worker         |   File "/app/superset/initialization/__init__.py", line 104, in __call__
superset_worker         |     return task_base.__call__(self, *args, **kwargs)
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 704, in __protected_call__
superset_worker         |     return self.run(*args, **kwargs)
superset_worker         |   File "/app/superset/tasks/scheduler.py", line 43, in scheduler
superset_worker         |     active_schedules = ReportScheduleDAO.find_active(session)
superset_worker         |   File "/app/superset/reports/dao.py", line 225, in find_active
superset_worker         |     session.query(ReportSchedule).filter(ReportSchedule.active.is_(True)).all()
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3373, in all
superset_worker         |     return list(self)
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3535, in __iter__
superset_worker         |     return self._execute_and_instances(context)
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3560, in _execute_and_instances
superset_worker         |     result = conn.execute(querycontext.statement, self._params)
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1011, in execute
superset_worker         |     return meth(self, multiparams, params)
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 298, in _execute_on_connection
superset_worker         |     return connection._execute_clauseelement(self, multiparams, params)
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1130, in _execute_clauseelement
superset_worker         |     distilled_params,
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1317, in _execute_context
superset_worker         |     e, statement, parameters, cursor, context
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1511, in _handle_dbapi_exception
superset_worker         |     sqlalchemy_exception, with_traceback=exc_info[2], from_=e
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
superset_worker         |     raise exception
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1277, in _execute_context
superset_worker         |     cursor, statement, parameters, context
superset_worker         |   File "/usr/local/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 608, in do_execute
superset_worker         |     cursor.execute(statement, parameters)
superset_worker         | sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation "report_schedule" does not exist
superset_worker         | LINE 2: FROM report_schedule 
superset_worker         |              ^
superset_worker         | 
superset_worker         | [SQL: SELECT report_schedule.created_on AS report_schedule_created_on, report_schedule.changed_on AS report_schedule_changed_on, report_schedule.id AS report_schedule_id, report_schedule.type AS report_schedule_type, report_schedule.name AS report_schedule_name, report_schedule.description AS report_schedule_description, report_schedule.context_markdown AS report_schedule_context_markdown, report_schedule.active AS report_schedule_active, report_schedule.crontab AS report_schedule_crontab, report_schedule.creation_method AS report_schedule_creation_method, report_schedule.timezone AS report_schedule_timezone, report_schedule.report_format AS report_schedule_report_format, report_schedule.sql AS report_schedule_sql, report_schedule.chart_id AS report_schedule_chart_id, report_schedule.dashboard_id AS report_schedule_dashboard_id, report_schedule.database_id AS report_schedule_database_id, report_schedule.last_eval_dttm AS report_schedule_last_eval_dttm, report_schedule.last_state AS report_schedule_last_state, report_schedule.last_value AS report_schedule_last_value, report_schedule.last_value_row_json AS report_schedule_last_value_row_json, report_schedule.validator_type AS report_schedule_validator_type, report_schedule.validator_config_json AS report_schedule_validator_config_json, report_schedule.log_retention AS report_schedule_log_retention, report_schedule.grace_period AS report_schedule_grace_period, report_schedule.working_timeout AS report_schedule_working_timeout, report_schedule.created_by_fk AS report_schedule_created_by_fk, report_schedule.changed_by_fk AS report_schedule_changed_by_fk 
superset_worker         | FROM report_schedule 
superset_worker         | WHERE report_schedule.active IS true]
superset_worker         | (Background on this error at: http://sqlalche.me/e/13/f405)
`

How can I run superset appropriately? What was my error?

**Comment by fehlerteufel** [comment]
I'm having the same problem on an Debian 11 machine.

**Reply by fehlerteufel** [reply]
Removing all docker volumes and checking out tag v2021.40.0 works though, so I assume it's not an error in my system config.
